### DISCUSSION TITLE
Does a 'Neutral' check run conclusion satisfy a required check?

### DISCUSSION THREAD

**Discussion by davidstaheli** [community/community#24754] [Apps, API and Webhooks]
<p>Check runs can complete with a <a href="https://developer.github.com/v3/checks/runs/" rel="nofollow noopener">Neutral</a> conclusion.  Does that conclusion satisfy a required check?  In other words, since it’s not a failure, can a PR with a required check be merged with a <strong>Neutral</strong> conclusion?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>As it states in the <a href="https://help.github.com/articles/about-required-status-checks/" rel="nofollow noopener">required status checks documentation</a>:</p>
<blockquote>
<p>After enabling required status checks, <strong>all required status checks must pass</strong> before branches can be merged into the protected branch.</p>
</blockquote>
<p>This means that all required checks must succeed.</p>
<p>I hope that helps!</p>

**Reply by cDima** [reply]
So neutral checks are considered a successful check, and PRs can be merged with a check that's completed as neutral.

**Reply by rohankurane23** [reply]
@cDima - was yours a question or a statement ? I am running a neutral check on my PR and would like to confirm. Thanks.

**Reply by Ilovegettinpaid88420** [reply]
> yes

**Comment by davidstaheli** [comment]
<p>Thank you, <a class="mention" href="/lee-dohm">@lee-dohm</a>.  Elsewhere I was told something slightly different:<br>
“success” and “neutral” are OK to deploy and merge.  “failure” and “error” are not.</p>

**Comment by eyarz** [comment]
<p>checked this scenario today (May 5, 2020) and the answer is <strong>YES</strong>.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/49474b50e50c5848696cdb2ae6f745daa233964a.png" data-download-href="/uploads/short-url/asfBlT6zyT8mBGJi02FDZTYhMD0.png?dl=1" title="2020-05-11 10_47_31-Update README.md by eyarz · Pull Request #14 · datree-demo_check-codeowners.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/19731161/181086584-e7553285-4ce6-4cf5-b220-e309f202e236.png" alt="2020-05-11 10_47_31-Update README.md by eyarz · Pull Request #14 · datree-demo_check-codeowners.png" data-base62-sha1="asfBlT6zyT8mBGJi02FDZTYhMD0" width="690" height="285" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/49474b50e50c5848696cdb2ae6f745daa233964a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2020-05-11 10_47_31-Update README.md by eyarz · Pull Request #14 · datree-demo_check-codeowners.png</span><span class="informations">765×317 20.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by larshp** [comment]
<p>Hmm, I read through the documentation today, it is still unclear to me if “skipped” and/or “neutral” will satisfy the required checks</p>
